hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeCreateRequest.java

```diff
@@ -117,21 +116,17 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
     boolean acquiredUserLock = false;
     IOException exception = null;
     OMClientResponse omClientResponse = null;
-    OmVolumeArgs omVolumeArgs = null;
-    Map<String, String> auditMap = new HashMap<>();
-    try {
-      omVolumeArgs = OmVolumeArgs.getFromProtobuf(volumeInfo);
-      // when you create a volume, we set both Object ID and update ID.
-      // The Object ID will never change, but update
-      // ID will be set to transactionID each time we update the object.
-      omVolumeArgs.setObjectID(
-          ozoneManager.getObjectIdFromTxId(transactionLogIndex));
-      omVolumeArgs.setUpdateID(transactionLogIndex,
-          ozoneManager.isRatisEnabled());
-
-
-      auditMap = omVolumeArgs.toAuditMap();
+    OmVolumeArgs omVolumeArgs = OmVolumeArgs.getFromProtobuf(volumeInfo);
```

Comment:
getFromProtobuf has the chance to throw exceptions, so shall we keep into the try scope?